packages/react-events/src/dom/Tap.js

```diff
@@ -80,10 +60,31 @@ type TapGestureState = {|
   y: number,
 |};
 
-type TapEvent = $ReadOnly<{|
+type TapState = {|
+  activePointerId: null | number,
+  buttons: 0 | 1 | 4,
+  gestureState: TapGestureState,
+  ignoreEmulatedEvents: boolean,
+  initialPosition: {|x: number, y: number|},
+  isActive: boolean,
+  pointerType: PointerType,
+  responderTarget: null | Element,
+  rootEvents: null | Array<string>,
+  shouldPreventClick: boolean,
+|};
+
+type TapEventType =
```

Comment:
Looks like it's because objectAssign is any
https://github.com/facebook/react/blob/master/packages/shared/ReactDOMTypes.js#L68
You need something like this
```
objectAssign: typeof Object.assign
```

Reply:
Flow doesn't like that either